### PR TITLE
Added checking for the bitmap of the interested level values

### PR DESCRIPTION
**Pull request by Mak-Di**
| Q | A |
| --- | --- |
| Is bugfix? | yes |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | yes |
- [x] Bug fix
- [x] Update docs
- [x] Test

### CODE REVIEW

File: framework/log/Target.php [hunk 1]
```diff
@@ -189,6 +189,12 @@ public function setLevels($levels)
                 }
             }
         } else {
+            $bitmapValues = array_reduce($levelMap, function ($carry, $item) {
+                return $carry | $item;
+            });
+            if (!($bitmapValues & $levels) && $levels != 0) {
+                throw new InvalidConfigException("Incorrect bitmap value : $levels");
```
Comment (samdark) [inline framework/log/Target.php]: I think it's better to say `Incorrect $levels value.` according to method signature.

Comment (Mak-Di) [inline framework/log/Target.php]: Done

**Comment by Mak-Di** [comment]
@samdark , Ready to review

File: framework/log/Target.php [hunk 2]
```diff
@@ -189,6 +189,12 @@ public function setLevels($levels)
                 }
             }
         } else {
+            $bitmapValues = array_reduce($levelMap, function ($carry, $item) {
+                return $carry | $item;
+            });
+            if (!($bitmapValues & $levels) && $levels != 0) {
```
Comment (samdark) [inline framework/log/Target.php]: `!= 0` → `!== 0`

Comment (Mak-Di) [inline framework/log/Target.php]: @samdark , Done

**Comment by samdark** [comment]
Merged with minor adjustments. Thanks!

**Pull request closed without merging**